spring-security/src/test/java/com/sap/cloud/security/spring/token/authentication/JwtDecoderBuilderTest.java

```diff
@@ -39,7 +39,7 @@ void withHttpClient() {
 
 	@Test
 	void buildHybridWithoutConfiguration_IllegalArgumentException() {
-		assertThrows(IllegalArgumentException.class, () -> cut.build());
+		assertThrows(IllegalStateException.class, () -> cut.build());
 	}
 
 	@Test
```

Comment:
I know it is not part of this PR, but why this test case is named `buildHybridWithoutConfiguration()` when both IAS and Xsuaa configs are provided?